src/app/components/shared/project-list-hover/project-list-hover.component.html

```diff
@@ -1,17 +1,9 @@
+<app-search-project (changeFilterProject)="filterProjects = $event"></app-search-project>
 <ul class="list-group content-projects">
-  <li
-    class="list-group-item list-group-item-action d-flex justify-content-between align-items-center"
-    *ngFor="let item of projects; let i = index"
-    (mouseenter)="showButton = i"
-    (mouseleave)="showButton = -1"
-    (click)="clockIn(item.id)"
-    [ngClass]="{ active: selectedId === item.id }"
-  >
-    {{ item.name }}
-    <span
-      *ngIf="showButton === i && selectedId !== item.id"
-      class="badge badge-light"
-      >Clock In</span
+    <li class="list-group-item list-group-item-action d-flex justify-content-between align-items-center" *ngFor="let item of projects | filterProject:filterProjects;  let i = index" (mouseenter)="showButton = i" (mouseleave)="showButton = -1" (click)="clockIn(item.id)"
```

Comment:
Please, fix the indentation, remember that we are using an indentation based in 2 spaces.

Reply:
done